sefaria/settings.py

```diff
@@ -316,7 +316,7 @@
 # GLOBAL_INTERRUPTING_MESSAGE = None
 
 
-
+SENTRY_DSN = os.getenv("SENTRY_DSN", None)
```

Comment:
I believe this shoud be placed in the local settings file as templated by the helm chart? 
This would make it consistent with how we manage other similar variables and make the management of environment specific customization easier.